app-misc/anki/anki-24.06.3-r1.ebuild

```diff
@@ -964,16 +947,16 @@ src_install() {
 }
 
 pkg_postinst() {
-	[[ "${REPLACING_VERSIONS%-r*}" = '2.1.15' ]] && local FORCE_PRINT_ELOG=1
+	ver_test ${REPLACING_VERSIONS} -lt 24.06.3-r1 && local FORCE_PRINT_ELOG=1
 	readme.gentoo_print_elog
 	if use gui; then
 		xdg_pkg_postinst
 		optfeature "LaTeX in cards" "app-text/texlive[extra] app-text/dvipng"
 		optfeature "sound support" media-video/mpv media-video/mplayer
-		optfeature "recording support" "media-sound/lame[frontend] dev-python/PyQt$(usex qt6 6 5)[multimedia]"
+		optfeature "recording support" "media-sound/lame[frontend] dev-python/PyQt6[multimedia]"
 		optfeature "faster database operations" dev-python/orjson
-		use qt6 && optfeature "compatibility with Qt5-dependent add-ons" dev-python/PyQt6[dbus,printsupport]
-		use qt6 && optfeature "Vulkan driver" "media-libs/vulkan-loader dev-qt/qtbase[vulkan]
+		optfeature "compatibility with Qt5-dependent add-ons" dev-python/PyQt6[dbus,printsupport]
```

Comment:
```suggestion
		optfeature "compatibility with Qt6-dependent add-ons" dev-python/PyQt6[dbus,printsupport]
```

Reply:
`Qt5-dependent add-ons` is actually correct. If `ENABLE_QT5_COMPAT` is set at runtime Anki imports the [`qt5_compat`](https://github.com/ankitects/anki/blob/24.06.3/qt/aqt/qt/qt5_compat.py) module, which pulls in the stated runtime dependency.
Said module exports the PyQt5 API by creating aliases to PyQt6 among other things. This way, older Anki add-ons that rely on the PyQt5 API keep working with the Qt6 GUI.